frontend/src/query_builder/GuiQueryEditor.jsx

```diff
@@ -59,8 +61,8 @@ export default class GuiQueryEditor extends Component {
         }
     };
 
-    setQuery(dataset_query) {
-        this.props.setQueryFn(dataset_query);
+    setQuery(query) {
+        this.props.setQueryFn(query);
```

Comment:
I think it's preferable to leave the old names in this case because `dataset_query` suggests this represents the full definition of a query.  This is a tough thing about reading the code given that we overload the use of the word `query` but thus far the convention has been that `dataset_query` represents the full query and `query` is just the inner structured query.